check_http_json.rb

```diff
@@ -88,7 +108,7 @@ def hash_flatten(hash, delimiter, prefix = nil, flat = {})
 # http://nagiosplug.sourceforge.net/developer-guidelines.html#THRESHOLDFORMAT
 def nutty_parse(thresh, want, got, v, element)
     retval = 'FAIL'
- 
```

Comment:
Thanks for cleaning up the trailing whitespace and stuff all over the place. :+1: